### PR TITLE
Adjustments to tooltip text

### LINKED ISSUE #105: Adjustments to tooltip text
**What needs to be done? Why does it need to be done?**

The scope of information that we have included in the tooltips has grown significantly since we started, so I wanted to start by recapping my understanding of the logic rules for when each piece of text should be included in a tooltip:

* Info (only show if we have data)
  * **Source Info Text**
  * **Last Updated Text**
* Warning (only show if one of these is non-empty)
  * if there is no data for this metric on either tab: **Not Available Text**
  * else
    * if the most recent data point is outside range: **Too Stale Text** 
    * if data for this metric is newer on the other tab
      * if there is data for this metric on this tab: **Restricted Text**
      * else: **Other Frequency Text**
      * **Use Switch Text**

I think this mostly aligns with how it is currently implemented, but calling it out in case it is useful in simplifying any of the current logic.

We need to refine the text in the tooltips to handle cases where data is provided directly to CSG instead of pulled from a publicly available report. The changes needed for that are:

* Source Info:
  * if there is a report url:
    * "Sourced from ${source}'s public reports."
  * else: 
    * "Sourced from data provided to the CSG Justice Center by ${source}."
  * "Includes data for the following categories: ${categories}"
  * "(${report})" (without link if there is no report url)
* Too Stale:
  * "This data is available in a public report from" -> "This data is available from"
* Other Frequency:
  * "(Monthly|Annual) data is not available in a public report from" -> "(Monthly|Annual) data is not available from"

cc @hobuobi 

**Additional context**
Add any other context about the task here.

### PR DESCRIPTION
**Pull request by nasaownsky**
## Description of the change

Added CSG related adjustments to the tooltips.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #105 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Review by jessex — APPROVED** [review]
Looks good to me! @colincadams since you know the requirements here very well, can you do a quick check and merge this if all looks well?

File: src/utils/generateSourceText.js [hunk 1]
```diff
@@ -18,15 +18,20 @@
...
+    if (sourceCategories.length) {
+      return `Sourced from ${sourceName}'s public reports. Includes data for the following categories: ${sourceCategories.join(
+        ", "
+      )}`;
+    }
+    return `Sourced from ${sourceName}'s public reports`;
   }
-  return `Includes data from ${sourceName}'s public reports`;
+
+  return `Sourced from data provided to the CSG Justice Center by ${sourceName}`;
```
Comment (colincadams) [inline src/utils/generateSourceText.js:34]: We still want to include the categories for data provided directly to CSG, if there are any categories provided!

**Review by colincadams — CHANGES_REQUESTED** [review]
Just once comment on the categories, otherwise this looks perfect. Thank you @nasaownsky!

File: src/utils/generateSourceText.js (hunk 1 above)
Comment (nasaownsky) [reply src/utils/generateSourceText.js:40]: Done!

**Review by colincadams — APPROVED** [review]
This works, thank you!!

**Pull request merged**